File: CS/Form1.cs
using System;
using System.Windows.Forms;

using System.Diagnostics;
using System.Drawing.Imaging;
using DevExpress.XtraPrinting;
using DevExpress.XtraReports.UI;
// ...


namespace ExportToImageCS
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // A path to export a report.
            string reportPath = "c:\\Test.png";

            // Create a report instance.
            XtraReport1 report = new XtraReport1();

            // Get its Image export options.
            ImageExportOptions imageOptions = report.ExportOptions.Image;

            // Set Image-specific export options.
            imageOptions.Format = ImageFormat.Png;

            // Export the report to Image.
            report.ExportToImage(reportPath);

            // Show the result.
            StartProcess(reportPath);
        }

        // Use this method if you want to automaically open
        // the created Image file in the default program.
        public void StartProcess(string path)
        {
            Process process = new Process();
            try
            {
                process.StartInfo.FileName = path;
                process.Start();
                process.WaitForInputIdle();
            }
            catch { }
        }
    }
}

File: VB/Form1.vb
﻿Imports System
Imports System.Windows.Forms

Imports System.Diagnostics
Imports System.Drawing.Imaging
Imports DevExpress.XtraPrinting
Imports DevExpress.XtraReports.UI
' ...


Namespace ExportToImageCS
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
            ' A path to export a report.
            Dim reportPath As String = "c:\Test.png"

            ' Create a report instance.
            Dim report As New XtraReport1()

            ' Get its Image export options.
            Dim imageOptions As ImageExportOptions = report.ExportOptions.Image

            ' Set Image-specific export options.
            imageOptions.Format = ImageFormat.Png

            ' Export the report to Image.
            report.ExportToImage(reportPath)

            ' Show the result.
            StartProcess(reportPath)
        End Sub

        ' Use this method if you want to automaically open
        ' the created Image file in the default program.
        Public Sub StartProcess(ByVal path As String)
            Dim process As New Process()
            Try
                process.StartInfo.FileName = path
                process.Start()
                process.WaitForInputIdle()
            Catch
            End Try
        End Sub
    End Class
End Namespace